management/openapi-3.yaml

```diff
@@ -2020,4 +2020,7 @@ components:
     CreativeTemplateUpdate:
       $ref: './schemas/creative-template.yaml#/schemas/CreativeTemplateUpdate'
   securitySchemes:
-    $ref: './components/security-schemes.yaml#/components/securitySchemes'
+    ApiKeyAuth:
+      type: apiKey
+      in: header
+      name: X-Adzerk-ApiKey
```

Comment:
I don't understand why this is effectively any different from what we have :thinking: 

The "before" version pointed to [components/security-schemes.yaml](https://github.com/adzerk/adzerk-api-specification/blob/8c171c788452aa6ca85bd3c20c1ed32b713e1e78/management/components/security-schemes.yaml), which contains exactly this, as far as I can tell.